pandas/_libs/writers.pyx

```diff
@@ -16,8 +16,6 @@ cimport numpy as cnp
 from numpy cimport ndarray, uint8_t
 cnp.import_array()
 
-cimport util
```

Comment:
do you have a linter that catches this?

Reply:
We've discussed this a couple times and haven't found a good solution for this.  It's on my list to bring up with the group  at the sprint.

Reply:
right, just asking if you found a soln!